projects/extension/build.py

```diff
@@ -170,7 +170,7 @@ def install_py() -> None:
             for d in version_target_dir.glob(
                 "pgai-*.dist-info"
             ):  # delete package info if exists
-                shutil.rmtree(d)
+                shutil.rmtree(d, ignore_errors=True)
```

Comment:
I'm not convinced that this is the fix to the build issues we're seeing.

The error we're seeing is:

```
/pgai/.venv/lib/python3.11/site-packages/pgai-0.8.1.dev0.dist-info`: No such file or directory (os error 2)
Traceback (most recent call last):
```

`python_install_dir()` is hardcoded to `"/usr/local/lib/pgai"`, so this line (`shutil.rmtree`) operates on the glob of `/usr/local/lib/pgai/0.8.1-dev/pgai-*.dist-info`.

Do you have any evidence that this change fixes the build issue?

Reply:
No evidence, I just wanted to test this was the fix by running in CI. Forgot to add it as draft. 

Indeed I don't think this is the issue. Also the error is when running pytest command so it's unrelated to this other command. Closing